### DISCUSSION TITLE
drag knife support

### DISCUSSION THREAD

**Discussion by hulda12** [terjeio/ioSender#194] [Q&A]
Hello,
is a drag knife supported?
Where can i customize that?

greeting Dirk

**Comment by terjeio** [comment]
_File > Transform > Add drag knife moves_

 Note that I have not tested it yet - please provide feedback if you try it.

**Reply by hulda12** [reply]
is that included in the iossender?

**Reply by terjeio** [reply]
Yes, in the _File_ menu.

**Reply by terjeio** [reply]
Info about the parameters can be found [here](http://www.atelier-des-fougeres.fr/Cambam/Aide/Plugins/DragKnife.html) .

**Reply by hulda12** [reply]
"tranform" is greyed out

**Reply by hulda12** [reply]
I forgot to upload a file. Thanks for the help, I'll try it and report.

**Reply by terjeio** [reply]
> "tranform" is greyed out

It should only be greyed out when no file is loaded. Did you load any?
